cars/v1/vanilla/templates/config/jvm.options

```diff
@@ -64,6 +64,9 @@
 # result in less optimal vector performance
 20-:--add-modules=jdk.incubator.vector
 
+# Lucene 10 testing: apply MADV_NORMAL advice to enable more aggressive readahead
```

Comment:
What happens if this option is passed on earlier versions?  No effect?

Reply:
no effect.

Reply:
it's a system property - so benign if no one actually reads or operates on it.